lib/libbackscrub.cc

```diff
@@ -375,3 +379,25 @@ bool bs_maskgen_process(void *context, cv::Mat &frame, cv::Mat &mask) {
 	return true;
 }
 
+cv::Rect calcCropping(int cw, int ch, int vw, int vh) {
```

Comment:
May I suggest:
 * use the same variable names as per the prototype in the header, they are more meaningful
 * stick to our library naming convention so `bs_calc_cropping` or similar.

Reply:
I am lazy, but you are right.

Reply:
Done, not pushed